rcl/include/rcl/timer.h

```diff
@@ -51,6 +51,13 @@ typedef struct rcl_timer_on_reset_callback_data_s
   size_t reset_counter;
 } rcl_timer_on_reset_callback_data_t;
 
+/// Structure which encapsulates call times about a ROS Timer.
```

Comment:
this refers to `ROS 1` timer, right? i think this is complicated with simulation clock based timer as in ROS 2.

how about the following in general in ROS 2?

```suggestion
/// Structure which encapsulates timer information when called.
```